docs/docs/tutorial/chapter6/comments-schema.md

```diff
@@ -200,6 +276,31 @@ export const Comment = {
 }
 ```
 
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```javascript title="api/src/services/comments/comments.ts"
+import { db } from 'src/lib/db'
+
+export const comments = () => {
+  return db.comment.findMany()
+}
+
+export const comment = ({ id }) => {
```

Comment:
```suggestion
export const comment = ({ id }: {id: number}) => {
```

Reply:
Totally missed adding types to this snippet. Made this line look like this: `export const comment = ({ id }: Prisma.CommentWhereUniqueInput) => {`